### ISSUE TITLE
Missing payment state attribute in payment request model

### ISSUE THREAD

**Issue by joarobles** [pagos360/sdk-php-v0#1]
According to the _PhpDoc_ a `PaymentRequest` object has the following properties:

```
/**
 * Represents a PaymentRequest
 *
 * @property string $description
 * @property string $externalReference
 * @property string $firstDueDate
 * @property float $firstTotal
 * @property string $secondDueDate
 * @property float $secondTotal
 * @property string $payerName
 * @property string $payerEmail
 * @property string $backUrlSuccess
 * @property string $backUrlPending
 *
 * @package Pagos360\Model
 */
```

But in the official [docs](https://ayuda.pagos360.com/v2/reference#get_payment-request-id) there are more properties available, such as the payment `state` (which is needed to check, for instance, if the information provided by a webhook call is true).

Is there a way to ask for the payment state with the sdk?

**Comment by gonza4** [comment]
Hi Joaquin,

The PaymentRequest object properties were updated. Now when using the method retrievePaymentRequest(id) you will be able to obtain, also, the status of it.

Regards!

**Comment by ghost** [comment]
This issue has been fixed in version [0.1.2](https://github.com/pagos360/sdk-php/releases/tag/0.1.2)
